brightwind/analyse/analyse.py

```diff
@@ -29,10 +29,17 @@
            'basic_stats',
            'TI',
            'sector_ratio',
-           'calc_air_density']
+           'calc_air_density',
+           'calc_air_density_from_vapour_prs',
+           'calc_saturation_vapour_pressure_of_water_vapour',
+           'calc_vapour_pressure_from_dewpoint',
+           'calc_vapour_pressure_from_relative_humidity']
 
 #  Specific gas constant for dry air (J/K/kg or m2/K/s2) from ISO:2533-1975 Standard Atmosphere
-GAS_CONST_DRY_AIR = 287.05 
+GAS_CONST_DRY_AIR = 287.05
```

Comment:
this is the same constant reported on scale.py (same comment for AIR_DENSITY_LAPSE_RATE) , we should have it only in one place. Can you add a constant.py file maybe inside utils folder, move all scale pressure, scale temperature, scale/derived air density constants there (e.g. GAS_CONST_DRY_AIR, GAS_CONST_WATER, AIR_DENSITY_LAPSE_RATE etc..) and update relevant .pys to get constants from there?